WcaOnRails/app/jobs/dump_developer_database.rb

```diff
@@ -3,12 +3,18 @@
 class DumpDeveloperDatabase < SingletonApplicationJob
   queue_as :default
 
-  def perform
+  def perform(force_export: false)
     # Create developer database dump every 3 days.
     last_developer_db_dump = Timestamp.find_or_create_by(name: 'developer_db_dump')
-    if last_developer_db_dump.not_after?(3.days.ago)
-      Rake::Task["db:dump:development"].invoke
```

Comment:
I came to the same conclusion that Rake tasks probably shouldn't be invoked like this when I realised that you have to intentionally `reenable` them when calling them twice from the same process. I guess it only worked to build the dev dump through this routine on a three day cron schedule like this because of some supervisor shenanigans that spin up new processes(??)
Since the server is running in the same PID, you have to hijack Rake. With that being said, I'm not wild about refactoring the entire way we generate exports when the focus right now is to ditch PHP